flex-linux-setup/flex_linux_setup/templates/aui_webhook.ldif

```diff
@@ -130,4 +130,11 @@ objectClass: auiFeatures
 objectClass: top
 auiFeatureId: saml_idp_delete
 displayName: SAML IDP - Delete
-jansScope: https://jans.io/idp/saml.delete
\ No newline at end of file
+jansScope: https://jans.io/idp/saml.delete
```

Comment:
It should have kc_link permission

Reply:
for `saml_idp_delete` it should have `kc_link` permission? please confirm.